xapian-bindings/android/CROSS_COMPILE

```diff
@@ -0,0 +1,90 @@
+Instructions for cross compiling for android 
+============================================
+This file is aimed to help developers get started with cross compiling xapian for android.
+
+
+Prerequisites
+=============
+1)
+You will need to download android ndk from https://developer.android.com/ndk/downloads/index.html 
+for your respective development platform. Extract it and now using the make-standalone-toolchain.sh
+in the ndk make a customized toolchain from the command line by doing the following:
+
+export NDK=<path_to_ndk>
+$NDK/build/tools/make-standalone-toolchain.sh --toolchain=<target-arch> --platform=<target-android-api> --install-dir=<path_to_ndk_standalone_toolchain>
+
+<target-arch> is the architecture your standalone toolchain is going to target. eg. for ARM-based architecture can
+use arm-linux-androideabi-4.9. For more info refer https://developer.android.com/ndk/guides/standalone_toolchain.html
+
+<target-android-api> is the target Android platform/API level. eg android-16
+
+<path_to_ndk_standalone_toolchain> is the path where your standalone toolchain will be installed.
+
+2)
+As Xapian makes use of built-in UUID APIs and ndk does not have UUID APIs, we need to explicitly cross compile that as well. So download libuuid, inside the e2fsprogs package by doing following:
+
+git clone git://git.kernel.org/pub/scm/fs/ext2/e2fsprogs.git
+cd e2fsprogs/
+
+Now configure and install the library by doing:
+export $TOOLCHAIN=<path_to_ndk_standalone_toolchain>
+export PATH=$PATH:$TOOLCHAIN
```

Comment:
I'm working through this, to try to set it up for the buildbot.

There are no executable files in the `$TOOLCHAIN` directory, so this step doesn't seem useful.  Should this actually be:

```
export PATH=$PATH:$TOOLCHAIN/bin
```

I think then you shouldn't need to set all the variables below.